schemas/yield-aggregator/schema.json

```diff
@@ -0,0 +1,180 @@
+{
+    "schema": "Yield Aggregator (Leverage, Gambling, RWA, ALM, Liquidity/LP, index)",
+    "description": "Standard table definitions for yield aggregator protocols (can include, Leverage, Gambling, RWA, ALM, Liquidity/LP, index).",
+    "version": "1.0.0-alpha",
+    "tables": [
+      {
+        "label": "Pools",
+        "tableName": "misc_pools",
```

Comment:
should they be classified as `misc`?

if we are grouping them all under `misc` then we might need a category in the pools schema

Reply:
I haven't changed this from our internal table names defined in the FE.

It is a TODO item to make better names, but it hasn't been a priority yet